HARK/tests/test_econforgeinterp.py

```diff
@@ -1,3 +1,4 @@
+from operator import index
```

Comment:
I don't think this is being used anywhere; and the next line

Reply:
You are right, thanks. VScode has gotten into the habit of adding strange, unused imports. Not sure why.